optuna/_multi_objective.py

```diff
@@ -26,6 +56,12 @@ def _get_pareto_front_trials(study: "optuna.study.BaseStudy") -> List[FrozenTria
     return pareto_front
 
 
+def _get_pareto_front_trials(study: "optuna.study.BaseStudy") -> List[FrozenTrial]:
+    if len(study.directions) == 2:
+        return _get_pareto_front_trials_2d(study)  # Log-linear in number of trials.
+    return _get_pareto_front_trials_nd(study)  # Quadratic in number of trials.
```

Comment:
The non-2d scenario seems not to be covered by any test case in `test_study.py`. We may want to add new ones, since it's risky to leave the general logic not tested.

Reply:
Added a test case for 3d.